components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/connector/recaptcha/SSOLoginReCaptchaConfig.java

```diff
@@ -172,7 +173,7 @@ public CaptchaPostValidationResponse postValidate(ServletRequest servletRequest,
     @Override
     public String getName() {
 
-        return CONNECTOR_NAME;
+        return SSO_LOGIN_RECAPTCHA_CONNECTOR_NAME;
```

Comment:
won't this a breaking change ? as have kept the org wise config referring to the name

Reply:
org wise connector config should be stored with this name right ?

Reply:
This is only a variable name change. The value of the connector name remains the same.